wordpress-mobile/WordPress-Android
# Stats landscape and tablet view

Fixes #8785 
This PR fixes landscape view in the new stats. The logic is that once the width of the screen gets bigger than 528dp (512dp as the max column width + 16dp as margins), we show only the column in the middle that's 512dp wide. Once the screen width reaches 720dp (704dp + 16dp margins), we start showing 2 columns in a block that's 704dp wide. This is useful on bigger phones and tablets.

To test:
* Open Stats on a device with screen size between 528dp and 720dp
* Check that there is a centered list in the middle of the screen with margins on both sides
* The list had one column 
* The functionality shouldn't change

To test 2:
* Open Stats on a device with screen size bigger than 720dp
* Check that there is a centered list in the middle of the screen with margins on both sides  
* The list has 2 columns
* The first item spread across both columns
* The functionality shouldn't change
